app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -7,41 +7,41 @@ import { RouteComponentProps, useParams } from "@reach/router";
 import AppLayoutWithExperiment from "../AppLayoutWithExperiment";
 import { useAnalysis } from "../../hooks";
 import { Alert } from "react-bootstrap";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
 import LinkExternal from "../LinkExternal";
+import TableSummary from "../TableSummary";
+import TableResults from "../TableResults";
+import TableHighlights from "../TableHighlights";
+import TableOverview from "../TableOverview";
+import TableMetricPrimary from "../TableMetricPrimary";
+import TableMetricSecondary from "../TableMetricSecondary";
+import { AnalysisData } from "../../lib/visualization/types";
 
 const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
   const { slug } = useParams();
-  const { loading, error, result } = useAnalysis(slug);
+  const { loading, error, result: analysis } = useAnalysis(slug);
 
   return (
-    <AppLayoutWithExperiment
-      title="Results"
-      testId="PageResults"
-      sidebar={false}
-    >
+    <AppLayoutWithExperiment title="Results" testId="PageResults">
```

Comment:
There's already the `Results` subtitle below. Perhaps we can called this `Analysis` instead? I know this ruins the component labeling a bit since it's called `PageResults`.

Reply:
This makes me wonder if we should change the page component to be named `PageAnalysis` and make the URL `/analysis`! I'll change just the page title to "Analysis" for now.